src/app/components/badge/badge.ts

```diff
@@ -22,13 +23,26 @@ export class BadgeDirective implements AfterViewInit, OnDestroy {
         this._disabled = val;
     }
 
+    @Input('badgeSize') public get size(): BadgeSize {
```

Comment:
In p-badge component, we use "size" as input could you please rename badgeSize as "size" to make it unify for both directive and component?

Reply:
Hi, I updated the size `input`.